.github/dependabot.yml

```diff
@@ -17,6 +17,9 @@ updates:
       python-dependencies:
         patterns:
           - "*"
+        exclude-patterns:
+          - "opensearch-py"
+          - "kubernetes"
```

Comment:
can't we just tell dependabot to update the patch versions?

Reply:
you can do something like this - but i need to test the it work's with groups
```yaml
    ignore:
      - dependency-name: "opensearch-py"
        versions: "18.*"
```

Reply:
could be worth to try? Up to you if you want to try in this PR or in follow PR.

Reply:
k so let close this  for now i will work on some change